streamlit_app.py

```diff
@@ -460,6 +475,72 @@ def draw_embedded_tweets(state, container):
+
+
+def write_narrative_2(container):
+    global NARRATIVE
+    NARRATIVE = INTERACTIVE_CONTROL != "Manual"
+
+    if INTERACTIVE_CONTROL == 'Narrative Tweets':
+        adjust_control_panel(state="Florida")
+
+    container.write("Next, we pivot our attention to how people are talking about Covid-19 on social media platforms. For this, we use Twitter and look for the most popular terms on Covid-19 related Twitter feeds. We have just seen through the data that employment type is a major factor in susceptibility to Covid-19.  Hopefully, we see people talking about this on Twitter.  ")
+    container.write("We continue our analysis of Florida and turn to its Twitter feed from the summer of 2020. For the ease of reading, we’ll look at the most frequent terms from tweets in bar chart form. Select the **Narrative Tweets** radio button on the sidebar. Unsurprisingly, the word “work” appears in the top 50 most frequent words used in tweets posted from Florida. However, it is at a relatively low place: 23rd out of 50. When we look at the most frequent words from our global set of tweets, “work” is 22nd out of the top 50. Surely something so important should be the topic of more conversations. And if not, what are people more focused on?")
+    container.write("One thing that stands out is that the words “trump” and “realdonaldtrump” (The President’s Twitter handle) appear in the top 5 words from Florida tweets. On a global scale, “trump” appears in the top 10 words used. This seems to be a cry for leadership to get us through a trying time.  And it makes sense that we would want to unite behind a strong leader during a crisis.  Among the Florida tweets, we also see popular terms like “stayhome”, “quarantine”, and “time”.  It seems people understand that this is a long process that we will need to stayhome and quarantine to get through.  One of the most telling things when going through all the states, and again when we look at the Global tweets from across the world is that the most popular term is “People”.  It is easy to get caught up in the politics of Covid-19, but it is good to know that, when we look at the tweets as a whole, we can see that people know that we the “People” are in this together. ")
+    container.write("We hope you found this narrative compelling, or at the very least interesting. We have only scratched the surface of some of the popular opinions that can be explored and insights that can be drawn from our app. ")
+    container.write("We encourage you to ask your own questions and interact with our app further to see what new conclusions can be drawn from exploring the data. Some ideas include: What impact did the heat and summer months have on the number of cases? And is there a correlation between education level and the decision to wear a mask while out in public? See what myths and common beliefs you can help support or debunk.")
+    container.write("Happy Busting!  \n - Mythbusters")
```

Comment:
I think this get's rendered as markdown which means that "- Mythbusters" will look like
- Mythbusters

Reply:
Thanks! I changed it to italics now so it'll look like:

Happy Busting!
*Mythbusters*